src/plugins/ui_core_views/cell_evaluation/evaluator.ts

```diff
@@ -261,7 +261,7 @@ export class Evaluator {
         ? this.computeFormulaCell(cellPosition.sheetId, cell)
         : evaluateLiteral(cell.content, localeFormat);
     } catch (e) {
-      return createEvaluatedCell(e.value, localeFormat, e.message);
+      return createEvaluatedCell(e.value || CellErrorType.GenericError, localeFormat, e.message);
```

Comment:
```suggestion
      return createEvaluatedCell(e?.value || CellErrorType.GenericError, localeFormat, e?.message || "some default message");
```
it is possible to throw an undefined error as well